test/humanizer.js

```diff
@@ -225,4 +225,20 @@ describe('humanizer', () => {
     assert.equal(h(15600000), '4 h, 20 m')
     assert.equal(h(1000, { language: 'es' }), '1 segundo')
   })
+
+  it('accepts fallback languages', () => {
+    const h = humanizer()
+
+    assert.strictEqual(h(10000, { language: 'es', fallbacks: ['en'] }), '10 segundos')
+    assert.strictEqual(h(10000, { language: 'BAD', fallbacks: ['BAD', 'es'] }), '10 segundos')
+  })
+
+  it('should not resolve a language with an invalid configuration', function () {
+    const h = humanizer()
+
+    assert.strictEqual(h(10000, { language: 'es', fallbacks: 'en' }), '10 segundos')
+    // Default language is en
+    assert.strictEqual(h(10000, { fallbacks: ['es'] }), '10 seconds')
+    assert.strictEqual(h(10000, { fallbacks: [] }), '10 seconds')
```

Comment:
IMO, an empty `fallbacks` should be an error. Could you move that to `test/error.js`?